cpu/stm32l0/periph/i2c.c

```diff
@@ -243,7 +252,8 @@ int i2c_read_bytes(i2c_t dev, uint8_t address, void *data, int length)
     _start(i2c, address, length, I2C_FLAG_READ);
 
     /* read the data bytes */
-    _read(i2c, data, length);
+    if (_read(i2c, data, length) < 0)
```

Comment:
Please add opening and closing curly braces, even for one line blocks. Even if it's more verbose, this improves code readability.
I found this in other places of your changes.

Reply:
done